handbook/company/values.md

```diff
@@ -110,3 +110,20 @@ Our code and PRs are public, so lack of transparency is only obfuscation, not se
 ### Continuously grow
 
 You strive to continuously grow and learn by genuinely soliciting feedback early and often, and humbly reflecting on your past mistakes.
+
+We value growth at Sourcegraph in the form of continuous learning:
+
+- We actively seek out learning opportunities instead of avoiding them.
+- We identify gaps in our knowledge and share them openly and honestly.
+- We adopt a growth mindset.
+
+Here are some ways that we support learning, teaching, and knowledge-sharing at Sourcegraph:
```

Comment:
I'd add our 360º peer review cycles as a major point here, I think they're a big pillar of continuous growth and open feedback culture 🙂 

Also regular retrospectives (which we have a handbook page for too I think) which are a big part of continuously improving as teams and blameless culture.

Reply:
@cassiemelani

Reply:
Great idea @felixfbecker - I'll add some things here!